src/dbcp/metadata/data_warehouse.py

```diff
@@ -591,6 +591,7 @@
     Column("summer_estimated_capability_mw", Float),
     Column("supercritical_tech", String),
     Column("can_switch_oil_gas", Boolean),
+    Column("can_switch_when_operating", Boolean),
```

Comment:
I'm not entirely sure why switching to nightly required adding this column to the `pudl_generators` data warehouse table. It seems like this column has been present in the EIA 860 multifuel table for 6 months... might be worth more investigation if/when we have time.